.circleci/config.yml

```diff
@@ -1058,7 +1047,7 @@ jobs:
 
   installation-tests-carthage:
     executor:
-      name: macos-executor
+      name: macos-executor-large
```

Comment:
Is this still needed after https://github.com/RevenueCat/purchases-ios/pull/4184? I checked that it was included in this PR, but these were so close together, maybe it's not needed now?

Reply:
This was already included! I rebased pretty often 😁 It was definitely _faster_ but I think that medium executor that was being used just sometimes wasn't powerful/large enough and carthage will timeout when trying to find/use the xcode project 🫠